### DISCUSSION TITLE
How do I use the configuration file in yaml format when I use docker

### DISCUSSION THREAD

**Discussion by Fury76** [vectordotdev/vector#13410] [Q&A] [answered]
How do I use the configuration file in yaml format when I use docker

The example only show me use toml format

**Comment by spencergilbert** [comment, marked as answer]
Hi @Fury76, Vector will parse the file as needed so this should work:

```shell
docker run -v $PWD/vector.yaml:/etc/vector/vector.yaml:ro timberio/vector:0.22.3-debian -c /etc/vector/vector.yaml
```

However, if you want to enforce YAML configs you can swap the `-c` for `--config-yaml`.
